src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -112,25 +114,36 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
-    private _createIframe(containerId: string, profileId: string, testMode: boolean, style?: MonerisStylingProps): HTMLIFrameElement {
+    private _createIframe(containerId: string, profileId: string, testMode: boolean, initializationData: any, style?: MonerisStylingProps): HTMLIFrameElement {
```

Comment:
Why are we using `any` as initialization data? can we do better than that and provide some type that better matches what we expect?